docs/manual/developer_guide.adoc

```diff
@@ -73,11 +76,22 @@ image::Patching-Pull_Requests.png[align="left"]
 image::Patching-Create_Pull.png[align="left"]
 
 4) You will be requested to input a patch title and description. Be concise, but thorough enough for a community member to understand logic behind your changes. Paste into the description field testing evidence (e.g. running testoval.py on any submitted OVAL, or before/after for remediation scripts).
+
+If you work on a feature or a bugfix that has an associated issue, either
+
+- assign yourself to the issue (if you have rights), or
+- mention the issue number in the pull request.
+
+This will improve the odds that multiple people won't work on a single issue without being aware of each other's work.
 After completing the form, select "Send pull request":
 
 image::Patching-Send_Pull_Request.png[align="left"]
 
-5) A community member will review your patch. They will either merge the patch upstream, indicate additional changes/documentation needed, or decline the patch. You'll automatically be notified via EMail of any status updates.
+5) Keep historical records clean - don't use git history-altering magic in your branch during your work on the pull request.
```

Comment:
`Keep historical records clean`
What do you mean? Be verbose here.

Reply:
I have rephrased that, but I think that the following text elaborates what was meant by that.